CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+# Contributing Guide
+
+This repo leverages the CircleCI _orb-tools_ orb as part of the [Orb Development Kit](https://circleci.com/docs/2.0/orb-author/#orb-development-kit). It provides a full suite of jobs for packing, validating, reviewing, and testing and deploying the orb to the orb registry.
+
+## Development
+
+The repo provides a subset of the orb-tools orb jobs and scripts can be ran locally. It is useful to be able to lint, shellcheck, and review your orbs locally, before committing. With this setup, it is possible to test your code locally, but integration tests of the built orb will be ran on CircleCI.
+
+**Note:** A dev build of the orb will be published as part of the `orb-tools/publish` job run on every commit. You can use it to test your changes during development. It is also used to do the integration tests as part of the `./.circleci/test-deploy.yml` workflow.
+
+### Local Packing
+
+All CircleCI orbs are single YAML files, typically named `orb.yml`. However, for development, it is often easier to break the code up into more manageable chunks. The `circleci orb pack` command, a component of the [orb development kit](https://circleci.com/docs/orb-development-kit/), is used to "pack" or condense the seprate YAML files together into a single `orb.yml` file.
+
+```bash
+circleci orb pack ./src
+```
+
+### Local Validating
+
+Once the `orb.yml` file has been generate via _packing_ you can now validate the configuration using the CircleCI CLI.
+
+```bash
+circleci orb validate orb.yml
+```
+
+### Local Linting
+
+The orb-tools orb's `orb-tools/lint` job uses a utility [yamllint](https://yamllint.readthedocs.io/en/stable/), which can be downloaded and ran locally, or you can invoke the job locally with the CircleCI CLI.
+
+Assuming you `./circleci/config.yml` file appears similar to the one in this repository, you will have imported the orb-tools orb and defined the `orb-tools/lint` job in a workflow. Using the CLI from this directory, use the following command to locally lint your orb:
```

Comment:
Is being in this directory required or is that just the simplest?

Also, a couple of tweaks to streamline it a bit...

```suggestion
Assuming your `./circleci/config.yml` file appears similar to the one in this repository, you will have imported the orb-tools orb and defined the `orb-tools/lint` job in a workflow. Use the Circle CLI from this directory with the following command to locally lint your orb:
```